aten/src/ATen/native/cpu/UnaryOpsKernel.cpp

```diff
@@ -55,8 +55,9 @@ template <typename T>
 void VmlLog(int64_t N, const T* X, T* Y) {
   constexpr int64_t K = Vectorized<T>::size();
   at::parallel_for(0, N, K, [=](int64_t begin, int64_t end) {
+    using VT = vec::vec_scalar_t<T>;
```

Comment:
Looks unrelated to reductions

Reply:
similar to above, since `map<BFloat16>` would require vec type to be `Vectorized<float>`, this would lead to compilation error if un-changed.